keyboards/novelkeys/nk87/keymaps/via/config.h

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2022 Kirill Malgichev (@makirill)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Backlight options */
+
+#define RGB_BACKLIGHT_ENABLED 1
```

Comment:
Just a heads up, via v3 supports QMK's per key rgb implementation, so could completely remove the via/wt rgb.

Reply:
This is something that I am planning to implement as a separate PR, just to keep changes smaller and focused.